build.gradle

```diff
@@ -3,12 +3,13 @@ plugins {
     id 'java-library'
     id 'maven-publish'
 
+    id 'com.diffplug.spotless' version '5.10.2'
     id 'com.github.ben-manes.versions' version '0.33.0'
-    id 'com.github.spotbugs' version '4.5.1'
-    id 'com.diffplug.gradle.spotless' version '3.27.2'
     id 'com.jfrog.artifactory' version '4.13.0'
     id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+    id 'jacoco'
```

Comment:
should this be up with the other non-versioned IDs? Should those be alphabetized as well?